misc.go

```diff
@@ -589,6 +589,7 @@ var tokenMap = map[string]int{
 	"BINDING":                  binding,
 	"BINDINGS":                 bindings,
 	"EXPR_PUSHDOWN_BLACKLIST":  exprPushdownBlacklist,
+	"DISABLED_OPTIMIZE_LIST":   disabledOptimizeList,
```

Comment:
Better to keep the same naming pattern. How about `OPT_RULE_BLACKLIST`?

Reply:
It contains expression push down and other things like cascade rules in the future besides logical rules. Shall we also change the name?

Reply:
I prefer to let this table to only contain the disabled rules. This the PR can be cherry-picked to release-3.0 and don't need to worry about the compatibility problems.

Reply:
OK, if that's so. I should change it.